### DISCUSSION TITLE
Count number of lines in a Paragraph

### DISCUSSION THREAD

**Discussion by schneyer** [dolanmiu/docx#1408] [Q&A] [answered]
Hi, is there a way to measure or count the number of lines in a rendered Paragraph? I've got a Table with one Paragraph per cell, and want to fill one of the cells to match the exact number of lines that have already been rendered to the other cell.

**Comment by dolanmiu** [comment, marked as answer]
sorry, theres no way to actually get information out of a document like that

**Reply by schneyer** [reply]
All good, I'll figure out another way to approximate content length. Thanks so much for the quick response!!
